opennlp-tools/src/main/java/opennlp/tools/dictionary/serializer/DictionaryEntryPersistor.java

```diff
@@ -230,16 +244,17 @@ public static boolean create(InputStream in, EntryInserter inserter)
 
   /**
    * Serializes the given entries to the given {@link OutputStream}.
-   *
+   * <p>
+   * <b>Note:</b>
    * After the serialization is finished the provided
    * {@link OutputStream} remains open.
    *
-   * @param out stream to serialize to
-   * @param entries entries to serialize
+   * @param out The {@link OutputStream} to serialize to.
+   * @param entries The {@link Entry entries} to serialize.
    *
-   * @throws IOException If an I/O error occurs
-   * @deprecated Use
-   *     {@link DictionaryEntryPersistor#serialize(java.io.OutputStream, java.util.Iterator, boolean)} instead
+   * @throws IOException Thrown if IO errors occurred.
+   * @throws InvalidFormatException Thrown if parameters were invalid.
+   * @deprecated Use {@link DictionaryEntryPersistor#serialize(OutputStream, Iterator, boolean)} instead.
```

Comment:
Didn't check history but might be a candidate for removal via separate issue? wdyt?